packages/api/src/command/medical/ihe-result/create-ihe-result.ts

```diff
@@ -37,28 +52,27 @@ export async function handleIHEResponse({ type, response }: IHEResult): Promise<
   };
 
   switch (type) {
-    case IHEResultType.PATIENT_DISCOVERY: {
-      await createPatientDiscoveryResult(response);
```

Comment:
I didnt think this operation deserved its own file since it was like 3 LOC and all the other DB inserts were in this function.

Reply:
It's 3 lines now, and then when we need to create a `PatientDiscoveryResult` in another place we might end up duplicating this. Also, doing like this exposes the Model to an unrelated command.

I suggest we keep to the pattern and create a `createPatientDiscoveryResult` command.

Ditto for the ones below.

Reply:
resolved!